decorator.rb

```diff
@@ -0,0 +1,12 @@
+require_relative 'Nameable'
+
```

Comment:
Kindly make sure that the `nameable` file is required correctly. In your case, nameable is capitalized which is causing an error in your application.

![image](https://github.com/GhostEsso/School_library/assets/38088818/de5273a8-030b-4fd2-a17d-d4c551d82c22)